app/assets/stylesheets/_settings.scss

```diff
@@ -65,6 +65,7 @@ $article-tile-comments-bg--blue: $bay-of-many;
 $article-tile-bg--grey: $concrete;
 
 $article-tile-bg--mas: $atlantis;
```

Comment:
Yep no problem, your right, it was a very specific colour, I'll double check its not used anywhere and will remove and commit